### PR TITLE
[R20-1268] New content type for search listing content with split-out fields for ease of use

### PR DESCRIPTION
**Pull request by krakerag**
This PR has a new content type for core that will allow the creation of search listing pages for Ripple v2. Also contains the work done for https://digital-vic.atlassian.net/browse/R20-1271 in updating the header.

This content type has been tested in both reference site and content-vic, installs correctly in both and is fully available under site admin and the JSONAPI.

### CODE REVIEW

File: config/default/.htaccess [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Deny all requests from Apache 2.4+.
```
Comment (vincent-gao) [inline config/default/.htaccess:1]: We don't use Apache, so this directory could be deleted?

**Review by vincent-gao — APPROVED** [review]
hi @krakerag , it looks good to me. just have a minor question.

File: config/default/.htaccess (hunk 1 above)
Comment (krakerag) [reply config/default/.htaccess:1]: Oh this is probably just a stray file, I'm not even sure why it's included.

I'll remove it, thank you for spotting it.

Comment (krakerag) [reply config/default/.htaccess]: Removed, ty :)

**Pull request merged**